docker/Dockerfile.arm64

```diff
@@ -0,0 +1,58 @@
+FROM arm64v8/ros:humble as base
+
+# Arguments for building
+ARG USERID
+ARG USER
+
+# Setup environment
+ENV TERM linux
+ENV DEBIAN_FRONTEND noninteractive
+RUN echo 'debconf debconf/frontend select Noninteractive' | debconf-set-selections
+
+# Copy requirement files and install dependencies
+COPY docker/requirements.txt .
+RUN apt-get update && apt-get install --no-install-recommends -y $(cat requirements.txt)
+RUN rm requirements.txt
+
+# Create a user with passwordless sudo
+RUN adduser --uid $USERID --gecos "ekumen developer" --disabled-password $USER
+RUN adduser $USER sudo
+RUN echo '%sudo ALL=(ALL) NOPASSWD:ALL' >> /etc/sudoers
+RUN echo "export QT_X11_NO_MITSHM=1" >> /home/$USER/.bashrc
+USER $USER
+
+# Adds USER to dialout and plugdev group.
+# This is needed to access the serial ports, for further references check
+# the libserial documentation.
+RUN sudo usermod -a -G dialout $USER
```

Comment:
It fails here for me:

```sh
 => ERROR [base 10/16] RUN sudo usermod -a -G dialout jesus-silva                                                                                                                                       0.2s
------
 > [base 10/16] RUN sudo usermod -a -G dialout jesus-silva:
0.179 sudo: effective uid is not 0, is /usr/bin/sudo on a file system with the 'nosuid' option set or an NFS file system without root privileges?
------
Dockerfile.arm64:27
--------------------
  25 |     # This is needed to access the serial ports, for further references check
  26 |     # the libserial documentation.
  27 | >>> RUN sudo usermod -a -G dialout $USER
  28 |     RUN sudo usermod -a -G plugdev $USER
  29 |     
--------------------
ERROR: failed to solve: process "/dev/.buildkit_qemu_emulator /bin/sh -c sudo usermod -a -G dialout $USER" did not complete successfully: exit code: 1
```

What I usually do with this type of errors is to change the user later, so I don't need to use sudo on the commands.